### DISCUSSION TITLE
What does this flow option do?

### DISCUSSION THREAD

**Discussion by Edzi11** [OrcaSlicer/OrcaSlicer#2263] [Q&A]
![Screenshot 2023-09-29 105503](https://github.com/SoftFever/OrcaSlicer/assets/123626106/35c116d8-8561-457b-b92a-847922693e83)
Where it says flow ratio about bridge flow, am I supposed to match this to filament flow or leave it at one?

**Comment by igiannakas** [comment]
It reduces the flow over bridges. Typically as bridges extrude in mid air the extrude is thicker as it doesn’t have anywhere to squish to.

Reducing the flow slightly makes the bridge layer thinner and better matches the layer height of the surrounding layers. I’ve set mine to 95%.

**Reply by Edzi11** [reply]
I thought that was what bridge flow does? Are both flow ratio and bridge flow the same?

**Reply by igiannakas** [reply]
Oh my apologies I thought you meant bridge flow setting. The flow ratio is basically a multiplier on top of the filament flow ratio setting. Unecessary in my view and should always be 1.

**Reply by Edzi11** [reply]
okay thats kinda what i was thinking too. It doesnt make sense to have this here since you already have this option under the filament settings plus why have it next to bridge flow why not next to bottom/top surface flow. it confused me. appreciate the response!
